clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/OperationCode.java

```diff
@@ -85,6 +91,8 @@ public static OperationCode valueOf(byte value) {
         return REPLACE;
       case 6:
         return REPLACE_CONDITIONAL;
+      case 7:
+        return PUT_WITH_WRITER;
```

Comment:
Lets add test for the new operation added

Reply:
done